### ISSUE TITLE
[Linux] Drag & drop local file to CT

### ISSUE THREAD

**Issue by ciranus** [giuspen/cherrytree#1163]
Drag & drop `/home/user/image.png` file from a file manager to CT will create `file:///home/user/image.png` text (not a link).

Proposal: 
Drag & drop `/home/user/image.png` file to CT should create a link to image.png file, which name could be either `file:///home/user/image.png` or `/home/user/image.png`

**Comment by txe** [comment]
Well, a funny thing is that we don't have code to support drag&drop to CT. I tried to reproduce it on XFCE and windows, but couldn't. But it maybe some hidden mechanism in a widget we are using.

Also, there are other possibilities what to do with the given text
- just insert the text
- insert a link, as you suggested
- insert a file
- insert a picture
It's quite hard to decide what is the best.

**Comment by ciranus** [comment]
I just drag the file icon from Dolphin to CT in a KDE environment and get this `file:///filename` text.
Amazing if there is not code in CT for this.
A bit strange if drag&drop has no effect at all in XFCE and windows: maybe a future complaint.
No clear idea on final decision, although I would have naturally expected a link to the file (without any specific link panel operation).

**Comment by txe** [comment]
I think drag&drop images to CT will improve usability, so reopen the issue

**Comment by chrisroat** [comment]
I was researching note taking apps for Linux, and tried cherrytree.  I create plots using jupyter notebooks, and found that I cannot drag-and-drop the images I create.  An example is below (which I tested using Chrome browser)

1. Go to https://jupyter.org/try and select classic notebook
2. Create a cell (Insert > Insert Cell Above)
3. Type the following and press shift-enter (it creates a square with a color range):

```
!pip install numpy matplotlib
import matplotlib.pyplot as plt
import numpy as np
plt.imshow(np.arange(64).reshape((8,8)))
```

Neither of these works:

- Drag and drop the image into CherryTree.  In this case, I get the encoded image data as text (e.g. "data:image/png;base64,iVBORw0KGgoAAAAN..."
- Copy and Paste does nothing

**Comment by txe** [comment]
@chrisroat , your issue (Copy/Paste image from jupyter) will be fixed in 0.99.20.
